packages/schematics/angular/application/files/package.json

```diff
@@ -3,12 +3,12 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
+    "e2e": "ng e2e",
```

Comment:
Wrong order with `build`.

Reply:
@hansl . Since I deleted the fork. Made another PR with these changes #458